### ISSUE TITLE
Fix buttons appearance

### ISSUE THREAD

**Issue by XhmikosR** [pi-hole/AdminLTE#1084]
The Whitelist, Blacklist and Audit buttons do not use Bootstrap's classes so they look off. There might be more cases. Also while at it we might as well make sure we add `type="button"` and `aria-label`/`aria-hidden`, although `aria-hidden` seems to be missing from most FA icons.

I'm gonna try to tackle this in the next days.

**Comment by XhmikosR** [comment]
I believe this was fixed in v5.0.
